tests/test_update_wsdl.py

```diff
@@ -0,0 +1,146 @@
+from unittest import mock
+
+import pytest
+import requests
+
+from correios.update_wsdl import WSDL_PATH, update_wsdl
+
+
+class TestUpdateWsdl:
+
+    file_body = 'Whatever'
+
+    @pytest.fixture
+    def mock_logger(self):
+        with mock.patch('correios.update_wsdl.logger') as mock_logger:
+            yield mock_logger
+
+    @pytest.fixture
+    def mock_open(self):
+        with mock.patch(
```

Comment:
can you use `mock.patch` as a decorator instead of context manager? IMO this will  make the code  more readbable as there will be less blocks

Reply:
(same comment to the tests below)